packages/strapi/lib/commands/exportTemplate.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const { resolve, join, basename } = require('path');
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+
+// All directories that a template could need
+const DIRECTORIES = ['api', 'components', 'config', 'data'];
```

Comment:
Why do we need to copy the config ? this could favor some config leaking by mistakes if this isn't required I would avoid it

Reply:
The only reason is to copy the `config/functions/bootstrap.js` to get the seed script.  I could target the `config/functions` directory instead or the `bootstrap.js` file directly.

Reply:
I see the point but a user that would be using this command would most likely not have a seed script. The looks like a command that is only needed in the expansion squad :) I think this should be in another cli actually :)

Reply:
Ok yeah I see what you are saying.  Originally I was thinking as an internal tool, but I do think it would be great to expose this to the community so we can encourage them to make templates/starters. cc @remidej @malgamves

Reply:
love the idea!! This would be very useful for a lot of people should it be an option in the CLI!! Plus not to mention it would take the friction out of creating templates/starters. Def for opening it up to everyone